manual_correction.py

```diff
@@ -756,6 +756,10 @@ def main():
     # Perform manual corrections
     for task, files in dict_yml.items():
         if task.startswith('FILES'):
+            # Check if task is in suffix_dict.keys(), if not, skip it
+            if task not in suffix_dict.keys():
+                logging.warning("WARNING: {} is not a valid task. Skipping it.".format(task))
+                continue
```

Comment:
As discussed in-person: this check is done after the `task.startswith('FILES')` check because the manual-correction script should ignore without warning keys that start with `CORR`, for example.

Reply:
Good point! Clarified in a comment in https://github.com/spinalcordtoolbox/manual-correction/pull/72/commits/93a38b2a3645313729fd8c7d8a988b18f4a1b9ff